app/routes/task_routes.py

```diff
@@ -1 +1,119 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.task import Task
+from ..db import db
+from datetime import datetime
+from app.models.task import Task
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+
+@tasks_bp.post("")
+def create_task():
+  request_body = request.get_json()
+
+  try:
+     new_task = Task.from_dict(request_body)
+
+  except KeyError:
+     response = {"details": "Invalid data"}
+     abort(make_response(response, 400))
+
+
+  db.session.add(new_task)
+  db.session.commit()
```

Comment:
Same comment as above in `goal_routes.py`, would prefer this logic to be in a helper function so you're not duplicating very similar code across different files.